editor/plugins/script_editor_plugin.h

```diff
@@ -95,7 +95,6 @@ class ScriptEditorBase : public Control {
 	virtual void tag_saved_version() = 0;
 	virtual void reload(bool p_soft) = 0;
 	virtual void get_breakpoints(List<int> *p_breakpoints) = 0;
-	virtual bool goto_method(const String &p_method) = 0;
```

Comment:
This became useless (I guess?) after @reduz added `Script::get_member_line(const String &p_member)`.